js/tests/redux/actions/SEMrushRequest.test.js

```diff
@@ -0,0 +1,65 @@
+import { CHANGE_COUNTRY, NEW_REQUEST, SET_REQUEST_SUCCEEDED,
+	SET_REQUEST_FAILED, SET_REQUEST_LIMIT_REACHED, NO_DATA_FOUND,
+	setSEMrushChangeCountry, setSEMrushNewRequest, setSEMrushRequestSucceeded,
+	setSEMrushRequestFailed, setSEMrushSetRequestLimitReached, setSEMrushNoResultsFound } from "../../../src/redux/actions/SEMrushRequest";
+
+describe( "SEMrushRequest actions", () => {
+	it( "returns a CHANGE_COUNTRY action with countryCode: nl when setSEMrushOpenModal is called with nl", () => {
+		const expected =  {
+			type: CHANGE_COUNTRY,
+			countryCode: "nl",
+		};
+		const actual = setSEMrushChangeCountry( "nl" );
+
+		expect( actual ).toEqual( expected );
+	} );
+	it( "returns a NEW_REQUEST action with countryCode: nl and keyphrase: " +
+		"yoast when setSEMrushNewRequest is called with nl and yoast", () => {
+		const expected =  {
+			type: NEW_REQUEST,
+			countryCode: "nl",
+			keyphrase: "yoast",
+		};
+		const actual = setSEMrushNewRequest( "nl", "yoast" );
+
+		expect( actual ).toEqual( expected );
+	} );
+	it( "returns a SET_REQUEST_SUCCEEDED action with { response: \"exampleresponse\" } " +
+		"when setSEMrushRequestSucceeded is called with { response: \"exampleresponse\" }", () => {
+		const expected =  {
+			type: SET_REQUEST_SUCCEEDED,
+			response: { response: "exampleresponse" },
+		};
+		const actual = setSEMrushRequestSucceeded( { response: "exampleresponse" } );
+
+		expect( actual ).toEqual( expected );
+	} );
+	it( "returns a SET_REQUEST_FAILED action with { response: \"exampleresponse\" } " +
+		"when setSEMrushRequestFailed is called with { response: \"exampleresponse\" }", () => {
+		const expected =  {
+			type: SET_REQUEST_FAILED,
+			response: { response: "exampleresponse" },
+		};
+		const actual = setSEMrushRequestFailed( { response: "exampleresponse" } );
+
+		expect( actual ).toEqual( expected );
+	} );
```

Comment:
I believe that these tests don't cover the 'real world' implementation; `response` should be an object, however that object does not have another `response` property within it, but a `results` object, including a `status` property.

In the case of `SET_REQUEST_FAILED`, we have an `error` property in the object, instead of a `response` property.